guides/common/modules/proc_setting-the-service-level.adoc

```diff
@@ -1,9 +1,9 @@
 [id="Setting_the_Service_Level_{context}"]
 = Setting the service level
 
-You can configure an activation key to define a default service level for the new host created with the activation key.
-Setting a default service level selects only the matching subscriptions to be attached to the host.
-For example, if the default service level on an activation key is set to Premium, only subscriptions with premium service levels are attached to the host upon registration.
+You can configure an activation key to define a default service level for the new host created with it.
```

Comment:
```suggestion
You can configure an activation key to define a default service level for the new host created with the activation key.
```
I recommend to revert this change. Otherwise, it might not clear from a quick read what "it" refers to (activation key? or service level? or host?).